### DISCUSSION TITLE
Can we use azure blob or mongodb as the storage.

### DISCUSSION THREAD

**Discussion by Sooraj577** [doccano/doccano#1810] [Ideas]
instead of uploading txt files or any other formats of data, can we use azure blob or mongodb as storage and get the text for annotation from the db using queries.

**Comment by david-engelmann** [comment]
Absolutely, if you check out the https://github.com/doccano/doccano-client repository, you'll find everything you need to automatically create projects, upload examples with annotations. More work would be involved if you wanted to switch out the doccano storage mechanism with an alternative but is very much possible!
